sim/target-agnostic.mk

```diff
@@ -39,9 +39,16 @@ PLATFORM ?=
 DRIVER_CC ?=
 DRIVER_H ?=
 
+platforms_dir := $(abspath $(firesim_base_dir)/../platforms)
+
+f1_CXX_FLAGS ?= -std=c++11 -I$(platforms_dir)/f1/aws-fpga/sdk/userspace/include
+f1_LD_FLAGS ?=
+
 # Target-specific CXX and LD flags for compiling the driver and meta-simulators
 TARGET_CXX_FLAGS ?=
+override TARGET_CXX_FLAGS += $($(PLATFORM)_CXX_FLAGS)
```

Comment:
Related to above. I think we should only add these when we are compiling a driver.

Reply:
Reverted.